reports/example.md

```diff
@@ -8,36 +8,46 @@ organisation: coala
 organisation_link : https://coala.io
 project: <project title>
 project_link: <project link>
+report_year: <year of completing GSoC>
 tarball: <tarball of patches' link>
+sha256sum: <sha256sum of tarball>
 mentors: >
- [mentor1]<link1> & [mentor2]<link2>
+  [mentor1]<link1> & [mentor2]<link2>
 phase:
- - Bonding : <bonding milstone link>
- - Phase 1 : <phase1 milsetone link>
- - Phase 2 : <phase2 milsetone link>
- - Phase 3 : <phase3 milsetone link>
+  - Bonding : <bonding milstone link>
+  - Phase 1 : <phase1 milsetone link>
+  - Phase 2 : <phase2 milsetone link>
+  - Phase 3 : <phase3 milsetone link>
 bio: >
- <your bio goes here>
+  <your bio goes here>
 social:
- - GitHub:
-   - username: <github username>
-   - link: https://github.com/<github username>
- - GitLab:
-   - username: <gitlab username>
-   - link: https://gitlab.com/<gitlab username>
- - Gitter:
-   - username: <gitter username>
-   - link: https://gitter.im/<gitter username>
- - HackerRank:
-   - username: <hackerrank username>
-   - link: https://hackerrank.com/<hackerrank username>
+  - GitHub:
+    - username: <github username>
```

Comment:
There should be 6 spaces before `- username`, not 4. Otherwise, YAMLlintbear will complain.
See https://docs.saltstack.com/en/latest/topics/troubleshooting/yaml_idiosyncrasies.html

Reply:
Wow, that is super weird. :P